__tests__/authkit-provider.spec.tsx

```diff
@@ -220,7 +220,6 @@ describe('useAuth', () => {
       entitlements: ['feature1'],
       impersonator: { email: 'admin@example.com' },
       oauthTokens: { access_token: 'token123' },
```

Comment:
we should probably remove these oauthTokens too for good measure (maybe someone else on the team thinks differently?).

Reply:
It looks like that's only referenced in that test and isn't actually passed to the client. We're only sending `UserInfo | NoUserInfo` from these actions https://github.com/workos/authkit-nextjs/blob/700f29d3b1521ba35fafb1ab0f556992f1f9c80e/src/interfaces.ts#L26-L45

Reply:
I went ahead and removed `oauthTokens` here from the mock return.

Reply:
oh i see, we pass the oauthTokens to an onSuccess callback but otherwise don't store them at all :+1: